mypy_primer/projects.py

```diff
@@ -144,6 +145,7 @@ def get_projects() -> list[Project]:
             mypy_cmd="{mypy} boostedblob",
             pip_cmd="{pip} install aiohttp uvloop pycryptodome",
             expected_mypy_success=True,
+            only_platform="linux",
```

Comment:
Would this be fine?:

```suggestion
            pip_cmd="{pip} install aiohttp pycryptodome",
            expected_mypy_success=True,
```